client/lib/cart-values/cart-items.js

```diff
@@ -605,6 +607,10 @@ function getDomainRegistrationsWithoutPrivacy( cart ) {
 	} );
 }
 
+function isPrivacyProduct( cartItem ) {
+	return cartItem.product_slug === 'private_whois';
+}
+
```

Comment:
The rest of the functions in the file have the same `function` syntax. I'd rather keep them in the same shape.